xkcd_get.py

```diff
@@ -0,0 +1,58 @@
+import requests
+import json
+import wget
+import logging
+import os
+import numpy as np
+from PIL import Image
+
+
+    
+
+def xkcd_get_img():
+    logging.info("Downloading xkcd-json")
+    response = requests.get("https://xkcd.com/info.0.json")
+    result = response.json()
+
+    logging.info("Downloading xkcd_img")
+    logging.info(result["img"])
+
+    path = 'path/to/your/waveshare/implementation'
+    filename = path + '/' + os.path.basename("xkcd-comic-strip.png")
+    if os.path.exists(filename):
+        os.remove(filename)
+    wget.download(result["img"], "xkcd-comic-strip.png")
+    
+    # Convert grayscale to pseudo-rgb since grayscale confuses the eink display
+
+    image = np.array(Image.open(filename).convert('RGB'))
+    Image.fromarray(image).save('xkcd-comic-strip.png')
+
+
+    """ If the downloaded image is too big to fit into the display downscale it
+    while taking the scales into account.
+    Afterwards resize the image to fit the screen. Sligh disortions can happen """
+    
+    im = Image.open(filename)
+    logging.debug("Png size: ",im.size)
+    if (im.size[0] > 800 and im.size[1] > 480):
```

Comment:
This didn't work for me because my screen size is smaller. You can tet the image height/width based on the `WAVESHARE_EPD75_VERSION` environment variable. (See run.sh for example)